app/Resources/public/css/print.css

```diff
@@ -119,3 +93,102 @@ table.table td {
     border: 1px solid #555;
     border-collapse: collapse;
 }
+
+/* print table pdf */
+.data_table{
+    width: 100%;
+    padding: 0;
+    border: 1px solid #b5cae7;
+    border-spacing: 0;
+    margin: 0 0 18px;
+}
+
+.data_table td {
+    padding: 5px;
+    border-bottom: 1px solid #b5cae7;
+    border-right: 1px solid #b5cae7;
+    border-top: 1px solid #b5cae7;
+    text-align: left;
+    font-size: 12px;
+    color: #666666;
+}
+
+.data_table th, .data_table td {
+    line-height: 18px;
+    padding: 10px;
+}
+.data_table th {
+    padding: 8px 10px;
+    font-weight: bold;
+    background-color: #eef5fa;
+    border-bottom: 2px solid #b5cae7;
+    color: #4171b5;
+}
+.data_table th a {
+    color: #333;
+}
+.data_table_no_border {
+    border-collapse: collapse;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)